Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -22,6 +22,8 @@ import de.rki.coronawarnapp.ui.viewmodel.SubmissionViewModel
 import de.rki.coronawarnapp.ui.viewmodel.TracingViewModel
 import de.rki.coronawarnapp.util.DialogHelper
 import de.rki.coronawarnapp.util.ExternalActionHelper
+import de.rki.coronawarnapp.util.di.AppInjector
```

Comment:
Why is this the Fragment in `.../deviceForTesters/...`?

Reply:
:rage: that's the split code issue again, good spot. Need to add it for both fragments `device` and `deviceForTesters`.

Reply:
It's about the file not about the line.